tests/modules/programs/vim-vint/basic-configuration.nix

```diff
@@ -0,0 +1,21 @@
+{ config, pkgs, ... }:
+
+{
+  config.programs.vim-vint = {
+    enable = true;
+    settings = {
+      cmdargs = {
+        severity = "error";
+        color = true;
+        env = { neovim = true; };
+      };
+      policies = {
+        ProhibitEqualTildeOperator.enabled = false;
+        ProhibitUsingUndeclaredVariable.enabled = false;
+        ProhibitAbbreviationOption.enabled = false;
+        ProhibitImplicitScopeVariable.enabled = false;
+        ProhibitSetNoCompatible.enabled = false;
+      };
+    };
+  };
+}
```

Comment:
The reason that the tests were passing earlier even though the expected configuration was wrong is that you did not add a test script! This test isn't actually testing anything. See other files in `tests/` for examples.

Reply:
Fixed in the latest push.

Reply:
You can run the test locally before pushing. `nix-shell --pure ./tests -A run.vim-vint-basic-configuration`